examples/sdk-migration-config.yaml

```diff
@@ -0,0 +1,206 @@
+# sdk-migration-config.yaml is a typical starting point for configuring the SDK when migrating
+# from environment variable based configuration.
+#
+# This template includes env var substitution references (i.e. ${MY_ENV}) for all spec defined
+# env vars (https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/)
+# which map cleanly to file configuration. For example, OTEL_SDK_DISABLED is referenced below,
+# but OTEL_TRACES_EXPORTER is not since it does not map well to the hierarchical structure of
+# file configuration.
+#
+# NOTE: With the exception of env var substitution syntax, SDKs ignore environment variables
+# when interpreting config files.
+#
+# Because env vars are ignored except for env var substitution, if "disabled: ${OTEL_SDK_DISABLED:-false}"
+# is replaced with "disabled: false", setting OTEL_SDK_DISALBED env var will not have any effect.
```

Comment:
This is circular logic.

> If "disabled: ${OTEL_SDK_DISABLED:-false}" is replaced with "disabled: false"

means that environment variable `OTEL_SDK_DISABLED` is not set, by definition.

Setting `OTEL_SDK_DISABLED` **will have some effect**, because it invalidates the conditions leading to the use of the default value.

Also, there is a typo on `OTEL_SDK_DISALBED`.

Reply:
Yes, upon rereading this phrasing is definitely confusing. I've reworded the phrasing.